src/Analysis/GWASApp/Components/Covariates/ContinuousCovariates.jsx

```diff
@@ -21,14 +21,22 @@ const ContinuousCovariates = ({
     concept_name: selected.concept_name,
   });
 
+  // when a user has selected a outcome phenotype that is a continous covariate with a concept ID,
+  // that should not appear as a selectable option, and be included in the submitted covariates.
+  // If they selected a outcome phenotype that is dichotomous
+  // the outcome doesn't need to be included as a submitted covariate
+  const submittedCovariateIds = outcome?.concept_id
+    ? [outcome.concept_id, ...selectedCovariates.map((obj) => obj.concept_id)]
+    : [...selectedCovariates.map((obj) => obj.concept_id)];
+
   return (
     <React.Fragment>
       <div className='GWASUI-flexRow continuous-covariates'>
         <div className='continous-covariates-table'>
```

Comment:
Actually it's here too, maybe we can get these updated?

Reply:
Good catch, thank you! Should be fixed now.